src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/StaticWhitelistTest.java

```diff
@@ -100,6 +100,10 @@ static void sanity(URL definition) throws Exception {
         for (EnumeratingWhitelist.Signature sig : sigs) {
             try {
                 assertTrue(sig + " does not exist (or is an override)", sig.exists());
+            } catch (AssertionError e) {
+                if (!KNOWN_GOOD_SIGNATURES.contains(sig)) {
+                    throw e;
+                }
```

Comment:
```suggestion
            if (KNOWN_GOOD_SIGNATURES.contains(sig)) {
                continue;
            }
            try {
                assertTrue(sig + " does not exist (or is an override)", sig.exists());
```
?

Reply:
Done in [e137c1f](https://github.com/jenkinsci/script-security-plugin/pull/394/commits/e137c1fb956e43a05583b32cc2f03b9ff7a8d9d2) along with a minor simplification to the catch statement for `ClassNotFoundException`.